### DISCUSSION TITLE
Libre Pro Sensor Activation Codes and How to fetch data from F-ram

### DISCUSSION THREAD

**Discussion by Biohacker17** [NightscoutFoundation/xDrip#2843] [Q&A]
Hello , I need a help for cracking Libre Pro activation Code and How to read F-Ram. I have recently integrated Freestyle Libre 1 sensor on my flutter project. Now i need help in integrating the Free Style Libre pro sensor.

**Comment by thulasirmn** [comment]
@Biohacker17 How you are integrated the flutter app with Libre 1 sensor can you post here i am also looking for the solution to integrate it

**Reply by Navid200** [reply]
Please don't use these forums for discussing reverse engineering.
These forums are meant to be used by xDrip users and developers to talk about bugs and feature requests for xDrip.
